arm/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -191,16 +191,16 @@ module azureFirewall_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, ind
 }]
 
 @description('The resource ID of the Azure firewall')
-output azureFirewallResourceId string = azureFirewall.id
+output resourceId string = azureFirewall.id
 
 @description('The name of the Azure firewall')
-output azureFirewallName string = azureFirewall.name
+output name string = azureFirewall.name
 
 @description('The resource group the azure firewall was deployed into')
-output azureFirewallResourceGroup string = resourceGroup().name
+output resourceGroupName string = resourceGroup().name
 
 @description('The private IP of the Azure Firewall')
```

Comment:
```suggestion
@description('The private IP of the azure firewall')
```

Reply:
Azure is lowercase? Would suggest `Azure firewall` instead

Reply:
updated, in general, the suggestion was more to align the description. It sometimes is lowercase (ref e.g. resourcegroupname description)